extensions/blocks/publicize/twitter/tweet-divider.js

```diff
@@ -201,6 +204,13 @@ export default compose( [
 							richTextIdentifier: container,
 							range: { start, end },
 						} );
+					} else if ( 'line-break' === type ) {
+						dispatch( 'core/annotations' ).__experimentalAddAnnotation( {
+							blockClientId: childProps.clientId,
+							source: 'jetpack-tweetstorm-line-break',
```

Comment:
Since this annotation requires different styling, it required adding a new `source`, since that's the only way we can control the `className` of the annotation. It's a little cumbersome (since it now requires checking for two annotation types), but I think it's the best option available.